config/initializers/inflections.rb

```diff
@@ -3,18 +3,12 @@
 # Add new inflection rules using the following format. Inflections
 # are locale specific, and you may define rules for as many different
 # locales as you wish. All of these examples are active by default:
-ActiveSupport::Inflector.inflections(:en) do |inflect|
-  # inflect.plural /^(ox)$/i, '\1en'
-  # inflect.singular /^(ox)en/i, '\1'
-  # inflect.irregular 'person', 'people'
-  # inflect.uncountable %w( fish sheep )
-
-  inflect.plural /(lea)f$/i, '\1ves'
-  inflect.singular /(lea)ves$/i, '\1f'
-  
-  inflect.plural /(canvas)$/, '\1es'
-  inflect.singular /(canvas)es/, '\1'
-end
```

Comment:
Don't we want these inflections?  They look ftp-specific to me.

Reply:
@saracarl I added those back.